Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -160,6 +161,12 @@
     <string name="analytics_new_id">Generate a New Statistics Identity</string>
     <string name="analytics_new_id_confirmation">Are you sure you want to generate a new statistics identity?</string>
 
+    <!-- SerialPort1 Subsetting Fragment -->
+    <string name="serialport1_submenu">Serial Port 1</string>
+    <string name="xlink_kai_guide_header">For setup instructions,\n<a href="https://www.teamxlink.co.uk/wiki/Dolphin">Refer to this page.</a></string>
```

Comment:
Refer shouldn't be capitalized. Unsure if we really should have a linebreak in this string.

Reply:
I went ahead and simulated a crappy resolution, I think even on lower resolutions the linebreak is not needed, so I'll remove it